docs/experiment/general/data-model.md

```diff
@@ -57,34 +59,34 @@ A variant exists within a flag or an experiment, and represents a variable exper
 
 Within Amplitude Experiment, users are passed to evaluation via `fetch` requests in [remote evaluation](evaluation/remote-evaluation.md), or directly to the `evaluate` function for [local evaluation](evaluation/local-evaluation.md).
 
 !!!warning
-    **Either a user ID or device ID must be included in the user object for evaluation to succeed.** E.g. remote evaluation will return a 400 error if both the User ID and Device ID are null, empty, or missing.
+    **Either a user ID or device ID must be included in the user object for evaluation to succeed.** For example, remote evaluation returns a 400 error if both the User ID and Device ID are null, empty, or missing.
 
 | <div class='med-big-column'> Property </div> | Type | Description |
 | --- | --- | --- |
-| `user_id` | `string` | The [User ID](https://help.amplitude.com/hc/en-us/articles/206404628-Step-2-Identifying-your-users#h_533ee533-f04d-49d3-873f-5836945f43a6) is the primary identifier for the user, generally when the user is logged into an account within your system. The User ID is used when resolving the Amplitude ID on enrichment  prior to [remote evaluation](evaluation/remote-evaluation.md) where the Amplitude ID is used the default bucketing key. |
-| `device_id` | `string` | The Device ID is the secondary identifier for the user, generally randomly generated by an analytics SDK on the client side or on the server-side and set in a cookie. The Device ID is also used when resolving the Amplitude ID on enrichment prior to [remote evaluation](evaluation/remote-evaluation.md) where the Amplitude ID is used the default bucketing key. |
-| `user_properties` | `object` | Optional object of additional custom properties to be taken into consideration when evaluating the user during local or remote evaluation.  |
-| `groups` | `object` | :material-beta: Optional object listing groups associated with this user. Format is an object where the key is the group type, and the value is an array of strings, the group value for the type (e.g. `{"org name":["Amplitude"]}`)  |
-| `group_properties` | `object` | :material-beta: Optional object listing group properties associated with this user. Format is an nested object where the key is the group type, and the value is an object where the key is a the group value, and the value is an object of properties (e.g. `{"org name":{"Amplitude":{"plan":"enterprise"}}}`) |
+| `user_id` | `string` | The [User ID](https://help.amplitude.com/hc/en-us/articles/206404628-Step-2-Identifying-your-users#h_533ee533-f04d-49d3-873f-5836945f43a6) is the primary identifier for the user, generally when the user is logged into an account within your system. The User ID is used when resolving the Amplitude ID on enrichment before [remote evaluation](evaluation/remote-evaluation.md) where the Amplitude ID is the default bucketing key. |
```

Comment:
@bgiori : "The User ID is the primary identifier for the user, generally when the user is logged into an account within your system:.

I'm finding this a little unclear. Does it mean:
"The User ID is the primary identifier for the user. This value typically comes from their user ID from your system" 

or does it mean something else?

Reply:
Yeah, thats more clear. "The User ID is the primary identifier for the user. This value is typically their user ID within your system."